modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -252,7 +252,7 @@ def track_526_validation_errors(errors)
         end
 
         def unprocessable_response(e)
-          log_message_to_sentry('Upload error in 526', :error, body: e.message)
+          log_message_to_sentry('Upload error in 526', :warning, body: e.message)
 
```

Comment:
I don't think we use this anywhere

Reply:
interesting. we don't really need to change this one anyway, but if it's unused ? 🤷